Demo/Podfile

```diff
@@ -12,6 +12,8 @@ def app_pods
   pod 'SVProgressHUD', '~> 2.1'
   pod 'AlamofireImage', '~> 3.2'
   pod 'AlamofireNetworkActivityIndicator', '~> 2.1'
+  pod 'RxSwift'
+  pod 'RxCocoa'
 end
 
 target 'Viper-v2-Demo' do
```

Comment:
Since we're on V4 now, this v2 naming is now wrong and should be updated throughout the project. In order to make our lives easier, let's remove the version completely and just have it as `Viper-Demo` 🙂